### PR TITLE
Migrate owner search tests

### PR DESCRIPTION
**Pull request by hennevogel**

### CODE REVIEW

File: src/api/spec/controllers/webui/search_controller_spec.rb [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+  let!(:develuser) { create(:confirmed_user, login: 'DevelIggy') }
+  let!(:package) { create(:package, name: 'TestPack', project: Project.find_by(name: 'home:Iggy')) }
+  let!(:develpackage) { create(:package, name: 'DevelPack', project: Project.find_by(name: 'home:DevelIggy')) }
+  let!(:owner_attrib) { create(:attrib, attrib_type: AttribType.where(name: "OwnerRootProject").first, project: Project.find_by(name: 'home:Iggy')) }
+
+  describe "GET #owner" do
+    it 'just returns with blank search text' do
+      get(:owner, { search_text: '', owner: 1 })
+      expect(response.status).to eq(200)
+    end
```
Comment (bgeuken) [inline src/api/spec/controllers/webui/search_controller_spec.rb]: Please add whitelines here

**Comment by bgeuken** [comment]
LGTM beside the missing whitespaces

File: src/api/spec/controllers/webui/search_controller_spec.rb [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+  let!(:user) { create(:confirmed_user, login: 'Iggy') }
+  let!(:develuser) { create(:confirmed_user, login: 'DevelIggy') }
+  let!(:package) { create(:package, name: 'TestPack', project: Project.find_by(name: 'home:Iggy')) }
+  let!(:develpackage) { create(:package, name: 'DevelPack', project: Project.find_by(name: 'home:DevelIggy')) }
+  let!(:owner_attrib) { create(:attrib, attrib_type: AttribType.where(name: "OwnerRootProject").first, project: Project.find_by(name: 'home:Iggy')) }
+
+  describe "GET #owner" do
+    it 'just returns with blank search text' do
+      get(:owner, { search_text: '', owner: 1 })
+      expect(response.status).to eq(200)
```
Comment (mdeniz) [inline src/api/spec/controllers/webui/search_controller_spec.rb]: I rather prefer that way:
```ruby
it { expect(response).to have_http_status(:success) }
```

**Comment by mdeniz** [comment]
LGTM beside my nitpick :+1:

**Pull request merged**